packages/api/src/external/aws/appConfig.ts

```diff
@@ -50,3 +54,19 @@ export async function isCQDirectEnabledForCx(cxId: string): Promise<boolean> {
   const cxIdsWithCQDirectEnabled = await getCxsWithCQDirectFeatureFlagValue();
   return cxIdsWithCQDirectEnabled.some(i => i === cxId);
 }
+
+export async function getPatientLimitForCx(cxId: string): Promise<number> {
```

Comment:
this is more metriport specific business logic as opposed to just getting the appConfig, and probably should be somewhere else?